runelite-client/src/main/java/net/runelite/client/plugins/banktags/BankTagsPlugin.java

```diff
@@ -205,6 +206,21 @@ else if (!tagSearch)
 				{
 					intStack[intStackSize - 2] = itemName.contains(search) ? 1 : 0;
 				}
+
+				if (config.priceFilter()) {
+					if (search.equals(PRICE_FILTER)) {
+						intStack[intStackSize - 2] = 1;
+					}
+					if (intStack[intStackSize - 2] == 1) {
+						ItemContainer container = client.getItemContainer(InventoryID.BANK);
```

Comment:
This should be cached on itemcontainerchanged

Reply:
Are you saying I should cache it or it's already cached?
How does this change what I'm doing? I still need to check the items individually and filter them.

Reply:
You are querying the bank item container for every item in the bank and then looping over it.